### DISCUSSION TITLE
Restrict access to IP addresses

### DISCUSSION THREAD

**Discussion by emrul** [neondatabase/neon#2068] [Ideas]
Is there any plan to restrict access to Neon databases to a list of static IP addresses (i.e. the source IPs for what would be my internal apps)?

**Comment by kelvich** [comment]
Hi Emrul,

yes, IP address restriction is on our roadmap.

**Reply by s3neotrue** [reply]
@kelvich 
I was simply wondering if there is any progress on this?
